src/foundry/common/data/data.mjs/ambientSoundData.d.ts

```diff
@@ -0,0 +1,181 @@
+import { ConfiguredFlags, FieldReturnType, PropertiesToSource } from '../../../../types/helperTypes';
+import DocumentData from '../../abstract/data.mjs';
+import * as documents from '../../documents.mjs';
+import { BaseAmbientSound } from '../../documents.mjs';
+import * as fields from '../fields.mjs';
+import { DarknessActivation, DarknessActivationConstructorData } from './darknessActivation';
+
+interface AmbientSoundDataSchema extends DocumentSchema {
+  _id: typeof fields.DOCUMENT_ID;
+  type: DocumentField<string> & {
+    type: typeof String;
+    required: true;
+    default: typeof BaseAmbientSound['metadata']['types'][0];
```

Comment:
```suggestion
    default: typeof documents.BaseAmbientSound['metadata']['types'][0];
```

(this is closer to foundry and also makes the import obsolete)